pkgs/tools/networking/croc/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "croc";
+  version = "6.1.1";
+
+  goPackagePath = "github.com/schollz/croc";
+
+  src = fetchFromGitHub {
+    owner = "schollz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "08gkwllk3m5hpkr1iwabvs739rvl6rzdnra2v040dzdj6zgyd12r";
+  };
+
+  modSha256 = "026m3hc2imna7bf4jpqm7yq6mr4l5is2crsx1vxdpr4h0n6z0v3i";
+  subPackages = [ "." ];
+
+  meta = with stdenv.lib; {
+    description     = "Easily and securely send things from one computer to another. Written in Go";
```

```suggestion
    description     = "Easily and securely send things from one computer to another";
```

No periods in `description`